### PR TITLE
Feature/api 7557 customers limits

### PR DESCRIPTION
**Pull request by czeczot**
## 🚀 Links

- [Feature branch] https://developers.labs.livechat.com/docs/feature/API-7557-customers_limits
- [Jira] https://livechatinc.atlassian.net/browse/API-7363

## ✅ Checklist (for API docs)

- Changelog
- API versions
- Web & RTM
- **Table:** Available methods/webhooks/pushes
- **Table:** Specifics

For more guidelines, see [Updating API docs](https://livechatinc.atlassian.net/wiki/spaces/PAT/pages/761529400/Updating+API+docs).

### Content

- New content
- Proofreading/content fixes
  
### Features (for the website)

- New feature
- Bug fix
- Breaking change

### CODE REVIEW

**Review by kogierman — APPROVED** [review]

File: content/management/limits/index.mdx [hunk 1]
```diff
@@ -24,6 +24,7 @@ Based on the <a href="https://www.livechat.com/pricing/" target="_blank">billing
 | Groups in group chooser       | 1       | 25   | 25       | 25         |
 | Number of banned visitors     | 50      | 200  | 400      | 400        |
 | Tracked visitors              | 400     | 400  | 400      | 400        |
+| Customers history             | 2m      | 1y   | 2y       | 4y         |
```
Comment (ale-kacperczyk) [inline content/management/limits/index.mdx:27]:
```suggestion
| Customers history             | 2 mos.   | 1 yr.| 2 yr.    | 4 yr.      |
```

File: content/management/limits/index.mdx [hunk 2]
```diff
@@ -34,6 +35,10 @@ There is a new type of error, `limit_reached`. Our APIs will return it when you
 - [List Archives](../../messaging/agent-chat-api/#list-archives)
 - [Ban Customer](../../messaging/agent-chat-api/#ban-customer)
 
+### v3.3
+
+- [List Customers][Web](/messaging/agent-chat-api/v3.3/#list-customers) & [RTM](/messaging/agent-chat-api/v3.3/rtm-reference/#list-customers)
```
Comment (ale-kacperczyk) [inline content/management/limits/index.mdx:40]:
```suggestion
- List Customers ([Web](/messaging/agent-chat-api/v3.3/#list-customers) & [RTM](/messaging/agent-chat-api/v3.3/rtm-reference/#list-customers))
```

**Review by ale-kacperczyk — CHANGES_REQUESTED** [review]

File: content/management/limits/index.mdx (hunk 1 above)
Comment (ale-kacperczyk) [reply content/management/limits/index.mdx:27]: Could you do the same for Chat archives history?

File: content/messaging/agent-chat-api/changelog/index.mdx [hunk 3]
```diff
@@ -51,6 +51,7 @@ This document is the record of all the changes in the **Agent Chat API**, Web an
 - The **List Customers** ([Web](/messaging/agent-chat-api/v3.3/#list-customers) & [RTM](/messaging/agent-chat-api/v3.3/rtm-reference/#list-customers)) method:
   - has a new parameter, `sort_by`.
   - has a new filter, `include_customers_without_chats`.
+  - has a new field `limited_customers` added to the response.
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/changelog/index.mdx:54]:
```suggestion
  - has a new field, `limited_customers`, added to the response.
```

**Review by ale-kacperczyk — APPROVED** [review]

**Pull request merged**